align/compiler/find_constraint.py

```diff
@@ -372,7 +372,8 @@ def process_symmnet(self):
                         elif key not in self.user_constrained_list:
                             self.user_constrained_list.append(key)
                             pairsj.append([key])
-                    if len(pairsj) > 0:
```

Comment:
@kkunal1408 , is there any reason to generate a symmetry placement constraint for a single instance? Below now makes sure this does not happen.

Reply:
@kkunal1408 , just a reminder on this.

Reply:
Sorry missed it: yes single pairs make sense e.g. [{MN1, MN2}] in a differential pair. The single self-symmetry instance does not make sense, e.g [{MN1}]

Reply:
yes just aligning vertically is the purpose, if it is creating a problem we can change this scenario to a align constraint.